gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+
+gem 'rubocop', '>= 1.0', '< 2.0'
```

Comment:
- [ ] Your `gemfile` is virtually empty. As such I cannot run `bundle install `to test your project. Kindly update your `gemfile`.

Reply:
Hi @t-yanick,

Upon checking it is working perfectly on my end.

### Here is the screenshot of it:

<img width="743" alt="Screenshot 2023-03-10 at 9 18 10 PM" src="https://user-images.githubusercontent.com/110017001/224367947-237f6b68-69e9-4a6d-8ab7-ec2520595648.png">